packages/react-examples/src/react-label/Label/Label.stories.tsx

```diff
@@ -55,60 +56,67 @@ export const CustomizableLabelExample = () => {
   const [required, setRequired] = React.useState(false);
   const [size, setSize] = React.useState<LabelProps['size']>('medium');
 
-  const updateLabelText = (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, val?: string) => {
-    if (val) {
-      setLabelText(val);
-    }
+  const updateLabelText = (ev: React.FormEvent<HTMLInputElement>) => {
+    setLabelText(ev.currentTarget.value);
   };
```

Comment:
It doesn't look like you're using `labelText` anywhere, or did I miss it?

Reply:
Actually a bunch of these state values don't appear to be used (`strong`, `disabled`, etc.)